data_platform_helpers/advanced_statuses/handler.py

```diff
@@ -298,8 +328,8 @@ def _on_status_detail_action(self, event: ActionEvent) -> None:
         event.set_results(
             {
                 "json-output": {
-                    "app": self.json_output(current_app_statuses),
-                    "unit": self.json_output(current_unit_statuses),
+                    "app": json.dumps(app_statuses_by_components.model_dump(mode="json")),
+                    "unit": json.dumps(unit_statuses_by_components.model_dump(mode="json")),
```

Comment:
This is changing the output format which has been designed in the UX specification of advanced statuses.
I think we should make something way easier (one line change) which is
```
event.set_results(
    {
	"json-output": json.dumps({"app": self.json_output(current_app_statuses), "unit": self.json_output(current_unit_statuses)})
    }
)
```